util/linux_install.sh

```diff
@@ -68,6 +68,7 @@ elif grep ID /etc/os-release | grep -q 'arch\|manjaro'; then
 		avr-libc \
 		avr-gcc \
 		base-devel \
+        dfu-programmer \
```

Comment:
Just a minor thing, but this line should be indented with tabs instead of spaces, to match the rest of the file.